backend/app/core/geocoding_service.py

```
@@ -12,6 +12,7 @@
 from datetime import datetime, timedelta
 
 from app.core.database import SessionLocal
+from app.core.croatian_geo_db import croatian_geo_db, CroatianLocation
```

Ruff (0.12.2)

`app.core.croatian_geo_db.CroatianLocation` imported but unused

Remove unused import: `app.core.croatian_geo_db.CroatianLocation`

(F401)